test/docx/0_level_headers.native

```diff
@@ -1,52 +1,182 @@
-[Table ("",[],[]) (Caption Nothing
- [])
- [(AlignDefault,ColWidth 1.0)]
- (TableHead ("",[],[])
- [])
- [(TableBody ("",[],[]) (RowHeadColumns 0)
-  []
-  [Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    [Plain [Str "User\8217s",Space,Str "Guide"]]]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    [Plain [Str "11",Space,Str "August",Space,Str "2017"]]]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]
-  ,Row ("",[],[])
-   [Cell ("",[],[]) AlignDefault (RowSpan 1) (ColSpan 1)
-    []]])]
- (TableFoot ("",[],[])
- [])
-,Para [Str "CONTENTS"]
-,Para [Strong [Str "Section",Space,Str "Page"]]
-,Para [Str "FIGURES",Space,Link ("",[],[]) [Str "iv"] ("#figures","")]
-,Para [Str "TABLES",Space,Link ("",[],[]) [Str "v"] ("#tables","")]
-,Para [Str "SECTION",Space,Str "1",Space,Str "Introduction",Space,Link ("",[],[]) [Str "2"] ("#introduction","")]
-,Header 1 ("figures",["Heading-0"],[]) [Str "FIGURES"]
-,Para [Strong [Str "Figure",Space,Str "Page"]]
-,Para [Strong [Str "No",Space,Str "table",Space,Str "of",Space,Str "figures",Space,Str "entries",Space,Str "found."]]
-,Header 1 ("tables",["Heading-0"],[]) [Str "TABLES"]
-,Para [Strong [Str "Table",Space,Str "Page"]]
-,Para [Strong [Str "No",Space,Str "table",Space,Str "of",Space,Str "figures",Space,Str "entries",Space,Str "found."]]
-,Header 1 ("introduction",[],[]) [Str "Introduction"]
-,Para [Str "Nothing",Space,Str "to",Space,Str "introduce,",Space,Str "yet."]]
+Pandoc
```

Comment:
<img width="679" alt="Screenshot 2023-10-31 at 14 27 39" src="https://github.com/jgm/pandoc/assets/3169264/aa0665cd-32ac-41e4-87b9-1ba265ddb492">

Reply:
Not sure what to do with this. It is entirely logical this is being converted to a `Title` (meta) so I would prefer letting that meaning just sit in there and maybe change the example.

Reply:
Can you explain more fully?  Are you saying that the Title style is used for both the document title and the title of the table of contents section?